pkg/storage/endpoint/tso_keyspace_group.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/tikv/pd/pkg/storage/kv"
+	"go.etcd.io/etcd/clientv3"
+)
+
+// KeyspaceGroup is the keyspace group.
+type KeyspaceGroup struct {
+	ID       uint32 `json:"id"`
+	UserKind string `json:"user-kind"`
+	// TODO: add `Members` field
```

Comment:
maybe we should call it `Replicas`. Why do you think?

Reply:
Both are ok to me.

Reply:
up to you then

Reply:
keep members then. I'm using members in my change.

Reply:
ok